crates/bevy_pbr/src/cluster/mod.rs

```diff
@@ -511,7 +512,8 @@ pub(crate) fn clusterable_object_order(
 /// Extracts clusters from the main world from the render world.
 pub fn extract_clusters(
     mut commands: Commands,
-    views: Extract<Query<(Entity, &Clusters, &Camera)>>,
+    views: Extract<Query<(&RenderEntity, &Clusters, &Camera)>>,
+    mapper: Res<MainToRenderEntityMap>,
```

Comment:
`MainToRenderEntityMap ` is used to manually correct the internal relationships dependencies between some components and other entities.the current implementation is quite ugly, perhaps we can have a dedicated system to correct these.

We use it here because the entity ID within the `Clusters `component is the entity ID of the light in the main world, and we must synchronize it with the ID of the render world

Reply:
I'm probably misunderstanding something, but doesn't `query_all_render_entity.get(main_entity)` and `mapper.get(main_entity)` evaluate to the same entity?

Reply:
yeah, you are right ,I misunderstood your meaning before.`MainToRenderEntityMap `is unnecessary